doc/rst/tools/mason/mason.rst

```diff
@@ -750,9 +754,19 @@ Submit a Package
 The mason registry will hold the manifest files for packages submitted by developers.
 To contribute a package to the mason-registry a chapel developer will need to host their
 package and submit a pull request to the mason-registry with the toml file pointing
-to their package. For a more detailed description follow the steps below.
+to their package. For a more detailed description follow the steps below. Publishing
+can be done with ``mason publish`` or manually.
 
-Steps:
+``mason publish`` Steps:
+      1) Write a library or binary package in chapel using mason
+      2) Host the package in a git repository. (e.g. GitHub)
+      3) Fork the mason-registry on GitHub
+      4) Ensure your package has a remote origin.
+      5) Run ``mason publish`` in your package
+      6) Go to the link provided to open a pull request to the mason registry.
+      7) Wait for mason-registry gatekeepers to approve PR.
+
+Manual Steps:
```

Comment:
I think we can probably take this out. Users should use the publish functionality. What do you think @ben-albrecht

Reply:
I think we should eventually remove this, but it might not be a bad idea to leave it in for 1 release cycle so that we can build more confidence in `mason publish`. This way, if a user runs into an issue with `mason publish`, they know what it's doing under the hood so that they can submit the package themselves manually.

Reply:
That was my thinking as well in leaving it

Reply:
Im ok with that!